### ISSUE TITLE
Can't punch without holding X on Dualshock 4 

### ISSUE THREAD

**Issue by Alphaw0lfxc** [VFansss/mgs2-v-s-fix#48]
For some reason I cant punch without holding the X button and I've checked the control settings and they say otherwise please respond as soon as possible. Thank you.

**Comment by VFansss** [comment]
Hi, sorry for the very late reply!

Are you using the DS4 through Steam or another external software? Is an original DS4? What operative system do you have?

Can you please sent me the padbtn.ini file you can find inside your game folder?

**Comment by Alphaw0lfxc** [comment]
I fixed it by updating the application that basically helps me use my DS4
on PC

On Aug 11, 2018 12:15 PM, "Alessio Di Giacomo" <notifications@github.com>
wrote:

Hi, sorry for the very late reply!

Are you using the DS4 through Steam or another external software? Is an
original DS4? What operative system do you have?

Can you please sent me the padbtn.ini file you can find inside your game
folder?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/VFansss/mgs2-v-s-fix/issues/48#issuecomment-412285038>,
or mute the thread

**Comment by VFansss** [comment]
Can you please tell me what application it is?

( I don't have a DS4 myself so I have to learn most things I can and write them to the wiki to help other players!)

**Comment by Alphaw0lfxc** [comment]
DS4Windows

**Comment by VFansss** [comment]
Just for curiosity: the controller is seen by the game if you don't use DS4Windows?

Or it is required to make it working?

**Comment by Alphaw0lfxc** [comment]
It's required

**Comment by Darthagnon** [comment]
> Just for curiosity: the controller is seen by the game if you don't use DS4Windows?
> 
> Or it is required to make it working?

Releases of DS4Windows can be found at https://github.com/Ryochan7/DS4Windows/releases
Wish I had a DS4, too - I currently use a fake USB DS2 and XOutput (https://github.com/csutorasa/XOutput/releases) to make games think it's an Xbox remote :D

**Comment by VFansss** [comment]
> Releases of DS4Windows can be found at https://github.com/Ryochan7/DS4Windows/releases

I didn't know that DS4Windows is now maintained by another person.

I will update the download link to the Fix Wiki and maybe I could insert it inside the fix itself, in the next version.

> XOutput (https://github.com/csutorasa/XOutput/releases) to make games think it's an Xbox remote :D

I didn't know about XOutput, I will test and add it to the wiki. It seems a much better alternative than x360CE!

> Wish I had a DS4, too - I currently use a fake USB DS2 and XOutput

Old school controllers are always the best 👍

**Comment by VFansss** [comment]
Now that I know better how is working the DS4, I'm more confident on how to aid users to play with it without issues.

I've edited the wiki to make more clear the situation about DS4, and also I've added a little message under the controller explanation ( on the 'Controls tab' ) to make more clear what would be right to choose.

Also, @Darthagnon make me know that fantastic tool named XOutput that is truly a gamechanger. I've added it to the wiki too.

Thanks guys 😄
